devdoc/docs/ContributeDocs.md

```diff
@@ -0,0 +1,197 @@
+
+By following these guidelines and including the required and optional
+components, you can create comprehensive documentation that empowers
+users and streamlines the adoption of your new software feature.
+### Who should write it?
+The boundary between a developer's responsibilities and the documentation team’s responsibility is not cast in stone. However, there are some guidelines that can help you decide what your responsibility is. Here are some examples:
+#### Changing the default value of a configuration setting
+This typically needs a simple one or two word change in the area of the documentation where that setting is documented. However, the change could impact existing deployments or existing code and therefore it might also require a short write-up for the Red Book and/or Release Announcement.
+If the setting is used by both bare-metal and containerized, you should provide information about how the new setting is used in each of those deployments.
+#### Adding or modifying a Language keyword, Standard Library function, or command line tool action
+This needs some changes to existing documentation so the best way to provide the information is in a documentation Jira issue.
+If it s a new keyword, function, or action, a brief overview should be included.
+For a Standard Library function, the developer should update the Javadoc comment in the appropriate ECL file. For a command line tool change, the developer should update the Usage section of the code.
+#### Adding a new feature that requires an overview.
+This is a candidate for either an MD file, a blog, or both. Since there should have been some sort of design specification document, that could easily be repurposed as a good start for this.  
+#### A feature/function that is only used internally to the system
+Since this is information that is probably only of interest to other developers, a write-up in the form of an MD file in the repo is the best approach. If it affects end-users or operations, then a more formal document or a blog might be a good idea. 
+If it affects existing deployments or existing code, then a Red Book notice might also be needed.
+#### Extending the tests in the regression suite
+New tests are frequently added and the regression suite readme files should be updated at the same time. 
+If the tests are noteworthy, we could add a mention in the Platform Release Notes.  
+### Placement
+In general, it makes sense to keep simple documentation near the code. For
+example, a document about ECL Agent should go in the ECLAgent folder.
+However, there are times where that is either not possible or a document
+may cover more than one component. In those cases, there are a few
+options as shown below.
+
+#### Other Folders
+
+**devdoc**: This is a general folder for any developer document.
+
+**devdoc**/**docs**: This is a folder for documents about documentation.
+
+**userdoc**: This is a collection of docs targeted toward the end-user
```

Comment:
Currently devdoc/userdoc.  Should it move?  Should devdoc and userdoc both be under doc?

Reply:
I like the folder structure we created before I got involved. This can change it later if we choose. 
I am adding full paths to folders in the doc (e.g., **devdoc**/**userdoc**/**blogs**)